components/raftstore/src/store/worker/pd.rs

```diff
@@ -2065,6 +2126,15 @@ where
         }
         if !self.slow_score.last_tick_finished {
             self.slow_score.record_timeout();
+            // If the last slow_score already reached abnormal state and was delayed for
+            // reporting by `store-heartbeat` to PD, we should report it here manually.
+            let now = UnixSecs::now();
+            let interval_second = now.into_inner() - self.store_stat.last_report_ts.into_inner();
+            if self.slow_score.should_force_report_slow_store()
+                && interval_second >= self.store_heartbeat_interval.as_secs()
```

Comment:
Should we use a new util fn `store_stat.store_heartbeat_delayed()` instead of:
`interval_second >= self.store_heartbeat_interval.as_secs()`?

Reply:
Another thinking:
Normally we take `2 * expected-time-duration` as delayed, is this `interval_second >= self.store_heartbeat_interval.as_secs()` too sensitive?

Reply:
> Another thinking: Normally we take `2 * expected-time-duration` as delayed, is this `interval_second >= self.store_heartbeat_interval.as_secs()` too sensitive?

Here, precisely, the above `2 * expected-time-duration` u mentioned is `2 * self.pd_store_heartbeat_tick_interval` ?
IMO, it's acceptable in the scenario where IO is hung. `self.store_heartbeat_interval` is kept same with `self.pd_store_heartbeat_tick_interval == secs(10)`, that is, the timeout threshold is `10s` in this case.